pkg/analysis/arrayofstruct/analyzer.go

```diff
@@ -119,7 +118,7 @@ func reportArrayOfStructIssue(pass *analysis.Pass, field *ast.Field) {
 		prefix = fieldName
 	}
 
-	message := fmt.Sprintf("%s is an array of structs, but the struct has no required fields. At least one field should be marked as %s to prevent ambiguous YAML configurations", prefix, markers.RequiredMarker)
+	message := fmt.Sprintf("%s is an array of structs, but the struct has no required fields. At least one field should be marked as required to prevent ambiguous YAML configurations", prefix)
```

Comment:
Do we need test updates with this change? I'd suspect we do.

Reply:
+1

Reply:
I don't think this change is enough because I can't see this message as `want` in test-cases yet. You just added valid case, but we also want invalid case of this message.